realTimeChat/server/helpers/index.js

```diff
@@ -0,0 +1,40 @@
+const usersList = [];
+
+const getIndividualRoomUsers = (room) => {
+  return usersList.filter((user) => user.room === room);
+};
+
+const AddNewUser = (user) => {
+  usersList.push(user);
+  return usersList;
+};
```

Comment:
This is more of a function and not a constructor/class, naming could be simple camel case

Reply:
You could have instead define a proper class/constructor to instantiate a new user. That would have been more readable